components/LoginForm/LoginForm.module.scss

```diff
@@ -0,0 +1,40 @@
+﻿$font-size: 60px;
+$font-family: sans-serif;
+$font-color: orange;
+
+.h2{
+  text-align: center;
+  font-size: $font-size/2;
+  color: $font-color;
+  font-family: $font-family;
+}
+
+.form{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.input{
+  color: $font-color;
+  background-color: white;
+  border: solid 2px $font-color;
+  font-family: $font-family;
+}
+
+.label{
+  color: $font-color;
+  font-size: $font-size/3;
+  font-family: $font-family;
+  margin-top: 10px;
+}
+
+.login{
+  background-color: $font-color;
+  border: none;
+  height: 40px;
+  width: 70px;
```

Comment:
I'd recommend using padding here instead of height/width - that means that modifying the text probably won't require us to modify the styles, because padding will just add a bit to the button around the text.